src/pages/MainPage/HeroSection.jsx

```diff
@@ -1,18 +1,22 @@
 import SearchPannel from '../../components/common/SearchPannel/SearchPannel';
+import BackgroundHero from '../../assets/img/mainPage/hero.png';
 import './heroSection.scss';
 
 const HeroSection = () => {
   return (
-    <section className="hero">
-      <div className="hero-wrapper">
-        <div className="hero-info">
-          <h1 className="hero-info-title">
+    <section
+      className="... min-h-[40.75rem] bg-cover bg-center bg-no-repeat"
```

Comment:
What is `...`?

Reply:
spread operator, lol

Reply:
@adadychanka  "All text have a white color".

At first I thought that you were talking about the fact that I can apply white color to the parent div-container. But then I took a closer look at your screenshot. On my site everything is fine with the text color.
![image](https://github.com/voyageglobal/voyage-frontend/assets/170460349/7478dc32-6ad2-496c-9009-e8f5f6059bcb)

Reply:
I've pulled a repo, run frontend and get while text color

Reply:
Okay, probably this is happening because we have different default colors in a browser.

Set the default color for the whole page and then override it for each text